changes/2854.feature.rst

```diff
@@ -0,0 +1 @@
+Add a write-only property in WebView to set the contents without an annoying empty string.
```

Comment:
Try to avoid pejorative terms in release notes.
```suggestion
The content of a WebView can now be assigned using the `content` property, without providing a root URL for the content.
```